src/core/WarpFactory.ts

```diff
@@ -67,14 +67,19 @@ export class WarpFactory {
    * with Warp testnet (https://testnet.redstone.tools/)
    */
   static forTestnet(
+    cacheOptions = defaultCacheOptions,
+    useArweaveGw = false,
     arweave = Arweave.init({
-      host: 'testnet.redstone.tools',
+      host: 'arweave.net',
```

Comment:
`validateEnv` method in `Testing.ts` class will not work properly, we should probably check warp env

Reply:
you're right - it will now work only in local testing env.

but....since we're using real arweave now for the 'testnet'...the `addFunds` method would not work either way :/

not sure how to handle this - this is a general issue with using mainnet as testnet...